internal/pkg/api/rest-handler.go

```diff
@@ -63,8 +63,6 @@ func (proxy *restProxy) handleV1DataGet(w http.ResponseWriter, r *http.Request)
 func (proxy *restProxy) handleV1DataPost(w http.ResponseWriter, r *http.Request) {
 	// Add unique identifier for logging purpose
 	r = utilInt.AssignRequestUID(r)
```

Comment:
I think we should either remove the request uid entirely or keep is somehow.
Because now we have the problem that the uid is logged inside the default-compiler if i.e. the request has a wrong format, but with not logging the initial request the uid becomes type of useless ;D

I think the best approach is to remove the entire uid thing from the incoming request header and log important fields like request URI including request body-fields like path or method inside the policy compiler itself.